pkg/brew/bfs.rb

```diff
@@ -0,0 +1,18 @@
+class Bfs < Formula
```

Comment:
Is this really the best place for files like this?  Does Homebrew not have its own collection of build scripts for other software?